modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -16,121 +16,73 @@ Traditional clients are not available on {centos}{nbsp}8.
 ====
 
 
-== Server Requirements
 
-Before you begin, check that your {productname} Server meets the requirements at xref:installation:hardware-requirements.adoc[].
+== Add Software Channels
 
-Taskomatic uses one CPU core, and requires at least 3072{nbsp}MB of RAM.
-To ensure that taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, and add this line:
+Before you register {centos} clients to your {productname} Server, check that you have the {centos} product enabled, and the required channels are fully synchronized.
 
-----
-taskomatic.java.maxmemory=3072
-----
+ifeval::[{suma-content} == true]
 
-Restart Taskomatic:
-----
-systemctl restart taskomatic
-----
+The channels you need for this procedure are:
```

Comment:
At the top we talk correctly about "products" but here we fallback to "channels". The Web UI only show Products.

Reply:
The UI calls them channels, though?

![image](https://user-images.githubusercontent.com/3914967/85359721-625a7180-b55a-11ea-99c1-2622bea7f8f3.png)

Reply:
Ok, when you explicitly click on the channel list.